trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
```

Comment:
You probably could talk about this a bit more.  While TC's are often thought of as highly opinionated gate-keepers.  Many people don't realize the responsibility they have.  I think it would be worth expanding this to talk about how they are commonly viewed in contrast to what they actually do and are responsible for.

Reply:
**1)** After the first sentence "The Trusted Committer (TC) role is one of the key roles in an InnerSource
community," there should be at least a one-line summary of what a TC is. Otherwise, someone who has no idea what it is,  won't hear the defintion until after the contrasting contributor role stuff has been covered.   **2)** A lot of great feedback has already been provided re: the TC role, but for what it's worth, here are some excerpts from notes that I had on the role: "The Trusted Committer (TC) has earned the responsibility of reviewing code contributions and handling merges. This role can require deep technical skills and a person who is well-versed in the specific technology. In addition, an important aspect of the TC role is that they must also have the capability and inclination to coach and mentor contributors, communicating in a clear and positive way. For example, rather than simply saying, 'This code is rejected,' an effective TC responds to a contributor by explaining the reason why the code cannot be merged as is, and providing guidance regarding what needs to happen for the code to be accepted. The TC role is a demanding role that requires both technical and interpersonal skills, as well as patience, and therefore does not suit everyone...It can be treated and viewed within an organization as an earned role within the career path, and one of esteem."

Reply:
> Care to elaborate on how TCs are viewed

I was thinking it would be useful to compare the role of TC to that of a shared internal project with someone running a project with very strong opinions.    

Using a foil like this would clarify why the TC role is unique and how the responsibility differs from other traditional roles like team lead, product owner or even developer.   It would also provide a relatable example to anyone trying to understand what to do different.

Reply:
e.g. Trusted Committer is **not** a dictator.  It is a mentor that wants to work with you to help you get your submission over the finish line.

Reply:
Thanks a lot @ErinMB , @mrsanz and @rrrutledge for expanding on this. I like Erins suggestion to add a one liner and introduce the TC role on a high level. Will give that a shot. Otherwise, the key difference of a TC to other roles to me is it's dual nature (tech _AND_ community oriented). That bit is already in the introduction, though.

@rrrutledge : I like your summary - I'll use that in the one-line-intro of the role.